prepare v2.24.0 for falcosidekick (#269)
falcosecurity/charts: falcosidekick/CHANGELOG.md

Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).

## 0.3.14

### Major Changes

* Add `Grafana` output
* Add `Fission` output
* Add `Yandex Cloud S3` output
* Add `Kafka REST` output

### Minor changes

> * Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`

Reviewer: ```suggestion
* Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`)
```

Author: fix

Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).

## 0.3.14

### Major Changes

> * New output: Grafana (PR#254)
> * New output: Fission (PR#255 thanks to @gauravgahlot)
> * New output: Yandex Cloud S3 (PR#261 thanks to @nar3k)
> * New output: Kafka REST (PR#263 thanks to @dirien)

Reviewer: ```suggestion
* Add `Grafana` output
* Add `Fission` output
* Add `Yandex Cloud S3` output
* Add `Kafka REST` output
```

Author: fixed

### Minor changes

> * Set header x-amz-acl to bucket-owner-full-control for output AWS S3 (PR#264 thanks to @Kaizhe)
> * Docker image is now available on AWS ECR Public Gallery (PR#265 thanks to @maxgio92)

Reviewer: ```suggestion
* Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`
```

Author: fixed
